test-infra/auto-deploy/deploy-master.yaml

```diff
@@ -20,8 +20,8 @@ spec:
       initContainers:
       - command:        
         - /usr/local/bin/checkout_repos.sh 
-        # Stop using PR #495 once its subbmitted
-        - --repos=kubeflow/kubeflow@HEAD,kubeflow/testing@HEAD:506
+        # Stop using PR #495 once its subbmitted        
```

Comment:
Should we remove this comment and not pin to `testing@HEAD:506` in this PR?

Reply:
Fixed